config/cloudrun/config.go

```diff
@@ -55,4 +55,12 @@ func Configure(p *config.Provider) {
 		}
 		config.MarkAsRequired(r.TerraformResource, "location")
 	})
+	p.AddResourceConfigurator("google_cloud_run_v2_job", func(r *config.Resource) {
+		// This prevents an import cycle not allowed error
+		r.References["template.template.vpc_access.connector"] = config.Reference{}
```

Comment:
```suggestion
		delete(r.References, "template.template.vpc_access.connector")
```

Both statements will do the same thing but you may be better off using the above configuration that we commonly use.

As an example: https://github.com/upbound/provider-gcp/blob/2876bb9642db87d510360133846b5aba142d7335/config/bigquery/config.go#L25